src/server/game/Spells/SpellMgr.cpp

```diff
@@ -4030,6 +4030,12 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->RecoveryTime = 20000;
     });
 
+    // Resurrection - Zul'Gurub, High Priest Thekal
+    ApplySpellFix({ 24173 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->CastTimeEntry = sSpellCastTimesStore.LookupEntry(1);
```

Comment:
add a comment with how much time is it (just to make life easy when someone read it)